python/example_code/glue/scaffold.py

```diff
@@ -0,0 +1,114 @@
+        setup_template = setup_file.read()
+    print(f"Creating {stack_name}.")
+    stack = cf_resource.create_stack(
+        StackName=stack_name,
+        TemplateBody=setup_template,
+        Capabilities=['CAPABILITY_NAMED_IAM'])
+    print("Waiting for stack to deploy.  This typically takes a minute or two.")
+    waiter = cf_resource.meta.client.get_waiter('stack_create_complete')
+    waiter.wait(StackName=stack.name)
+    stack.load()
+    print(f"Stack status: {stack.stack_status}")
+    print("Created resources:")
+    for resource in stack.resource_summaries.all():
+        print(f"\t{resource.resource_type}, {resource.physical_resource_id}")
+    print("Outputs:")
+    outputs = {}
+    for oput in stack.outputs:
+        outputs[oput['OutputKey']] = oput['OutputValue']
+        print(f"\t{oput['OutputKey']}: {oput['OutputValue']}")
+    return outputs
+
+
+def destroy(stack, cf_resource, s3_resource):
+    """
+    Destroys the resources managed by the CloudFormation stack, and the CloudFormation
+    stack itself.
+
+    :param stack: The CloudFormation stack that manages the demo resources.
+    :param cf_resource: A Boto3 CloudFormation resource.
+    :param s3_resource: A Boto3 S3 resource.
+    """
+    bucket_name = None
+    for oput in stack.outputs:
+        if oput['OutputKey'] == 'BucketName':
+            bucket_name = oput['OutputValue']
+    if bucket_name is not None:
+        print(f"Deleting all objects in bucket {bucket_name}.")
+        s3_resource.Bucket(bucket_name).objects.delete()
+    print(f"Deleting {stack.name}.")
+    stack.delete()
+    print("Waiting for stack removal.")
+    waiter = cf_resource.meta.client.get_waiter('stack_delete_complete')
+    waiter.wait(StackName=stack.name)
+    print("Stack delete complete.")
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Deploys and destroys scaffold resources for the 'Getting started "
+                    "with crawlers and jobs' scenario. Run with the 'deploy' action to "
+                    "deploy resources or with the 'destroy' action to destroy resources.")
+    parser.add_argument(
+        'action', choices=['deploy', 'destroy'],
+        help="Indicates the action the script performs.")
+    parser.add_argument(
+        '--script', default='setup_scenario_getting_started.yaml',
+        help="The name of the CloudFormation script to use to deploy resources.")
+    args = parser.parse_args()
+
+    print('-'*88)
+    print("Welcome to the AWS Glue getting started with crawlers and jobs scenario.")
+    print('-'*88)
+
+    cf_resource = boto3.resource('cloudformation')
+    stack = cf_resource.Stack('doc-example-glue-scenario-stack')
+
+    try:
+        if args.action == 'deploy':
+            print("Deploying scaffold resources for the demo.")
+            outputs = deploy(args.script, stack.name, cf_resource)
+            print('-'*88)
+            print("To run the scenario, pass it the role and bucket names: ")
```

Comment:
print("To run the scenario, pass the role and bucket names to it: ")

Reply:
fixed